nailgun/entities.py

```diff
@@ -4883,6 +4884,25 @@ def remove_ansible_role(self, synchronous=True, timeout=None, **kwargs):
             client.delete(path, **kwargs), self._server_config, synchronous, timeout
         )
 
+    def play_ansible_roles(self, synchronous=True, timeout=None, **kwargs):
+        """Play all assigned ansible roles on a Host
+
+        :param synchronous: What should happen if the server returns an HTTP
+            202 (accepted) status code? Wait for the task to complete if
+            ``True``. Immediately return the server's response otherwise.
+        :param timeout: Maximum number of seconds to wait until timing out.
+            Defaults to ``nailgun.entity_mixins.TASK_TIMEOUT``.
+        :param kwargs: Arguments to pass to requests.
+        :returns: The server's response, with all content decoded.
+        :raises: ``requests.exceptions.HTTPError`` If the server responds with
+            an HTTP 4XX or 5XX message.
+
+        """
+        kwargs = kwargs.copy()  # shadow the passed-in kwargs
+        kwargs.update(self._server_config.get_client_kwargs())
+        response = client.post(self.path('play_roles'), **kwargs)
```

Comment:
Can you include a unittest for this func/path?

Reply:
Thanks, Can you add play_ansible_roles to PathTestCase::test_id_and_which?

Reply:
Done